spec/spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,26 @@
 
```

Comment:
- **[OPTIONAL]** I don't think there is any use in creating another `spec` folder within the `spec` directory. Instead of `spec/spec/...`, I think you should leave it as `spec/...`